lib/action.ts

```diff
@@ -80,14 +110,81 @@ export const fileFeedEventFromAction = (action: Action): FileFeedEvent => {
 };
 
 const descriptionForEventType = (eventType: string) => {
-  switch (eventType) {
-    case FileFeedEventType.RecordsCreated:
-      return "New records were created in Flatfile.";
-    case FileFeedEventType.RecordsUpdated:
-      return "Records were updated in Flatfile.";
-    case FileFeedEventType.RecordsDeleted:
-      return "Records were deleted in Flatfile.";
-    default:
-      return "An unknown event occurred.";
-  }
+  eventType = eventType.replace("upload:completed", "file:created");
```

Comment:
I'd just cut the description all around. Now that we can't specify the events specifically the description isn't adding anything extra to the page

Reply:
Removed description and related types/vars